docs/stylesheets/main.css

```diff
@@ -114,6 +114,45 @@ body {
   font-weight: 400;
 }
 
+/* mobile sidebar */
+
+.sm-sidebar {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: var(--default-color);
+  color: var(--white);
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 6rem;
+}
+
+.sm-sidebar-open {
+  display: none;
+  cursor: pointer;
+  margin-left: auto;
+  font-size: 3.2rem;
+  padding: 2rem;
+  color: var(--white);
+}
+
+.sm-sidebar-close {
+  cursor: pointer;
+  position: absolute;
+  top: 0;
+  right: 25px;
+  font-size: 3.6rem;
+}
+
+.sm-sidebar-header {
+  background-color: var(--secondary-color);
+  color: var(white);

```

Comment:
Here is your variable name should be --white?

Reply:
Thanks for pointing out that Aishwarya.

Reply:
Fixed